src/plugins/controls/public/options_list/components/options_list_popover.tsx

```diff
@@ -52,12 +53,13 @@ export const OptionsListPopover = ({
   return (
     <div
       id={`control-popover-${id}`}
+      className={`optionsList__popover`}
       style={{ width: width > 300 ? width : undefined }}
       data-test-subj={`optionsList-control-popover`}
       aria-label={OptionsListStrings.popover.getAriaLabel(fieldName)}
     >
       <EuiPopoverTitle paddingSize="s">{title}</EuiPopoverTitle>
-      {field?.type !== 'boolean' && (
+      {field?.type !== 'boolean' && !hideSearch && (
```

Comment:
@Heenawter , you are right and I agree with changing the name to `hideActionBar`.

I was thinking if it makes sense to have a control to toggle the visibility of search bar. Do we want to go for a scenario where `search` panel is not visible and rest of 3 controls are visible. I guess that might seem weird. What do you think?

At least we do not have that requirement and I am good renaming it to `hideActionBar`

Reply:
I think hiding the entire top action bar makes sense to me! :) I assume if this is being done, the solution knows that there are < 10 options to select from. So searching, sorting, etc. shouldn't be necessary in these cases, since the options list size is manageable without them :+1:

Reply:
Done.